plugins/modules/sefcontext.py

```diff
@@ -99,8 +99,6 @@
     support: full
   diff_mode:
     support: full
-  platform:
-    platforms: rhel
```

Comment:
Please keep this; it will work once #5837 is merged (with the correct docs fragment added).

Reply:
(The platform name should be `RHEL` though probably, not `rhel`.)

Reply:
Modules `dnf`, `rpm_key`, `yum`, ... use lower case `rhel`. You sure about the capitalization?

Reply:
Also a minority of modules add `support: full` to even this attribute.

In general there's disappointing lack of documentation about this feature - even the PR and issue for it are void of ~anything useful. Perhaps I should put it to the agenda some week.

Reply:
I think it's not fully defined yet what exactly `support` means for `platform`. I'm also not sure what's the best venue to discuss this. @bcoca what do you think? Maybe you already have a good definition for this :) Also also a good idea about allowed/suggested/expected values for `platforms`.

Reply:
Added `platform` back with lower case "rhel" since that's what other modules use.

Reply:
I am not sure what I am doing wrong - the CI still fails even after a rebase:
```
00:26 ERROR: Found 2 validate-modules issue(s) which need to be resolved:
00:26 ERROR: plugins/modules/sefcontext.py:0:0: invalid-documentation: DOCUMENTATION.attributes.platform.description: required key not provided @ data['attributes']['platform']['description']. Got None (75%)
00:26 ERROR: plugins/modules/sefcontext.py:0:0: invalid-documentation: DOCUMENTATION.attributes.platform.support: required key not provided @ data['attributes']['platform']['support']. Got None (75%)
```

Is the syntax that I used wrong? I lifted the attributes syntax from `ansible.builtin` modules, is it different in c.g?
```
  platform:
    platforms: rhel
```

I am looking at the `attributes.py` but I'm not sure what to make of it. The syntax for `platform` looks very different from all the other attributes in the file.

Is it supposed to be just 1 line
```
  platform: rhel
```
?